packages/backend/server/src/__tests__/models/user.spec.ts

```diff
@@ -272,6 +272,9 @@ test('should trigger user.deleted event', async t => {
   t.true(
     spy.calledOnceWithExactly({ ...user, ownedWorkspaces: ['test-workspace'] })
   );
+  // await for 'user.deleted' event to be emitted and executed
```

Comment:
this is strange, this test will only create testing module instead of the whole app, there is no 'user.deleted' listener at all

Reply:
We can temporarily solve it first. Find the root cause after vitest is refactored. It might not even exist by then.